executor/common_fuchsia.h

```diff
@@ -216,26 +220,7 @@ long syz_future_time(long when)
 	default:
 		delta_ms = 10000;
 	}
-	zx_time_t now = zx_time_get(ZX_CLOCK_MONOTONIC);
+	zx_time_t now = zx_clock_get(ZX_CLOCK_MONOTONIC);
 	return now + delta_ms * 1000 * 1000;
 }
 #endif
-
-#if defined(SYZ_EXECUTOR) || defined(__NR_zx_channel_call_finish) || defined(zx_channel_call_noretry)
-#include "kernel/lib/vdso/vdso-code.h"
-#define UNEXPORTED(name) ((syscall_t)((long)&zx_handle_close - VDSO_SYSCALL_zx_handle_close + VDSO_SYSCALL_##name))
-#endif
-
-#if defined(SYZ_EXECUTOR) || defined(__NR_zx_channel_call_finish)
-zx_status_t zx_channel_call_finish(long a0, long a1, long a2, long a3, long a4, long a5, long a6, long a7, long a8)
-{
-	return UNEXPORTED(zx_channel_call_finish)(a0, a1, a2, a3, a4, a5, a6, a7, a8);
```

Comment:
What's wrong with testing them?
This usage does not fix fuchsia api. If/when they go away or change signatures, we can change syzkaller accordingly. But for now they just allow syzkaller to test a little bit more kernel code.

Reply:
syzkaller can't build with a vdso-code.h include right now: I'm not super-sure how this worked before, but, kernel/lib/vdso isn't part of the sysroot, and thus isn't currently accessible to a third-party binary.

There's certainly ways to thread it through, but I thought perhaps it made sense to get something that builds first, and then add back in private syscalls in a separate CL as we're so inclined.